src/Renderer/HorizonRenderer.cpp

```diff
@@ -41,36 +119,191 @@ HorizonRenderer::Draw(Canvas &canvas, const PixelRect &rc,
+  rotate(m1, center, Angle::Degrees(180 - bank_degrees), p1);
+  rotate(m2, center, Angle::Degrees(180 - bank_degrees), p2);
+  canvas.DrawLine(p1,p2);
+
+  m1.x = - radius + Layout::Scale(7);
+  rotate(m1, center, Angle::Degrees(70 - bank_degrees), p1);
+  rotate(m2, center, Angle::Degrees(70 - bank_degrees), p2);
+  canvas.DrawLine(p1,p2);
+  rotate(m1, center, Angle::Degrees(80 - bank_degrees), p1);
+  rotate(m2, center, Angle::Degrees(80 - bank_degrees), p2);
+  canvas.DrawLine(p1,p2);
+  rotate(m1, center, Angle::Degrees(100 - bank_degrees), p1);
+  rotate(m2, center, Angle::Degrees(100 - bank_degrees), p2);
+  canvas.DrawLine(p1,p2);
+  rotate(m1, center, Angle::Degrees(110 - bank_degrees), p1);
+  rotate(m2, center, Angle::Degrees(110 - bank_degrees), p2);
+  canvas.DrawLine(p1,p2);
+
+  m2.x = - radius + Layout::Scale(9);
+  rotate(m2, center, Angle::Degrees(45 - bank_degrees), p2);
+  canvas.DrawCircle(p2, 2);
+  rotate(m2, center, Angle::Degrees(135 - bank_degrees), p2);
+  canvas.DrawCircle(p2, 2);
+
+  PixelPoint m3 = {- radius + Layout::Scale(12),0};
+  m1 = {- radius + Layout::Scale(2), -5};
+  m2 = {- radius + Layout::Scale(2), 5};
+  rotate(m1, center, Angle::Degrees(90 - bank_degrees), p1);
+  rotate(m2, center, Angle::Degrees(90 - bank_degrees), p2);
+  rotate(m3, center, Angle::Degrees(90 - bank_degrees), p3);
+
+  BulkPixelPoint zero_mark[3] = {p1, p2, p3};
+  canvas.DrawPolygon(zero_mark, 3);
+
+  for (int k=-20; k<=20; k+=10){
+    m1 = {-radius/5, int(h/2.f*(pitch_degrees+k)/40)};
+    m2 = {+radius/5, int(h/2.f*(pitch_degrees+k)/40)};
+    rotate(m1, center, Angle::Degrees(- bank_degrees), p1);
+    rotate(m2, center, Angle::Degrees(- bank_degrees), p2);
+    canvas.DrawLine(p1,p2);
+  }
+  for (int k=-25; k<=25; k+=10){
+    m1 = {-radius/10, int(h/2.f*(pitch_degrees+k)/40)};
+    m2 = {+radius/10, int(h/2.f*(pitch_degrees+k)/40)};
+    rotate(m1, center, Angle::Degrees(- bank_degrees), p1);
+    rotate(m2, center, Angle::Degrees(- bank_degrees), p2);
+    canvas.DrawLine(p1,p2);
+  }
+
   // draw aircraft symbol
   canvas.Select(look.aircraft_pen);
-  canvas.DrawLine({center.x + radius / 2, center.y}, {center.x - radius / 2, center.y});
-  canvas.DrawLine({center.x, center.y - radius / 4}, {center.x, center.y});
-
-  // draw 45 degree dash marks
-  const int rr2p = uround(radius * M_SQRT1_2) + Layout::Scale(1);
-  const int rr2n = rr2p - Layout::Scale(2);
-  canvas.DrawLine({center.x + rr2p, center.y - rr2p},
-                  {center.x + rr2n, center.y - rr2n});
-  canvas.DrawLine({center.x - rr2p, center.y - rr2p},
-                  {center.x - rr2n, center.y - rr2n});
+  canvas.Select(look.aircraft_brush);
+  canvas.DrawLine({center.x + radius - radius/5, center.y}, {center.x + radius / 10, center.y});
+  canvas.DrawLine({center.x - radius + radius/5, center.y}, {center.x - radius / 10, center.y});
+  canvas.DrawLine({center.x + radius/10, center.y + radius / 10}, {center.x + radius/10, center.y});
```

Comment:
Again broken history. if you move lines around then changes to them are not apparent.